packages/@react-spectrum/buttongroup/test/ButtonGroup.test.js

```diff
@@ -137,7 +137,14 @@ describe('ButtonGroup', function () {
       };
       let {getByTestId} = render(<ButtonGroupWithRefs setUp={setUp} />);
       let buttonGroup = getByTestId(buttonGroupId);
-      expect(buttonGroup).not.toHaveAttribute('class', expect.stringContaining('spectrum-ButtonGroup--vertical'));
+
+      if (process.env.STRICT_MODE) {
```

Comment:
can we solve this with the util we worked on instead? I'd prefer the results be the same

Reply:
Hm a bit tricky due to how `setUp` is called in the useEffect in `ButtonGroupWithRefs` which means our mock implmementations aren't actually set until the first overflow measurements happen in ButtonGroup. Digging now

Reply:
I opted to remove the first expect that this test originally had since it was an assertion made before any of our mock widths even were applied (aka widths were undefined for the button/buttongroup). That felt a bit unrealistic for the test and would require me to set the mock widths to 0 for StrictMode to preserve that assertion's logic so I removed it instead